spec/models/element_spec.rb

```diff
@@ -580,6 +580,30 @@ module Alchemy
       end
     end
 
+    describe '#cache_key' do
+      let(:page) { stub_model(Page, published_at: Time.now - 1.week) }
+      let(:element) { stub_model(Element, page: page, updated_at: Time.now) }
+
+      subject { element.cache_key }
+
+      before do
+        expect(Page).to receive(:current_preview).and_return(preview)
+      end
+
+      context "when current page rendered in preview mode" do
+        let(:preview) { page }
+
+        it { is_expected.to eq("alchemy/elements/#{element.id}-#{element.updated_at}") }
+      end
+
+      context "when current page not in preview mode" do
+        let(:preview) { nil }
+
+        it { is_expected.to eq("alchemy/elements/#{element.id}-#{page.published_at}") }
+      end
+    end
+
+
```

Comment:
Oh, the first time @houndci is right \o/

@robinboening could you please remove the extra blank line? Thanks